4.4/4.4.1/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 4.4.1
+Создайте объект calculator (калькулятор) с тремя методами:
+
+read() (читать) запрашивает два числовых значения и сохраняет их в свойства (придумать название этим свойствам)
+текущего объекта.
+sum() (суммировать) возвращает сумму сохранённых значений.
+mul() (умножить) перемножает сохранённые значения и возвращает результат.
+А потом воспользоваться этим объектом, чтобы в консоль вывести сумму и произведение введенных числовых значение в
+формате:
+'Введенные числа: 5 и 6. Сумма - 11. Произведение - 30.'
+<script>
+	const calculator = {
+		'first-number': null,
```

Comment:
do you need '' here?
And in JS we use camel case in order to give a name to a variable or function